compiler/z/codegen/OMRCodeGenerator.hpp

```diff
@@ -1178,6 +1173,12 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
    TR::list<TR::S390ConstantDataSnippet*>  _constantList;
    TR::list<TR::S390ConstantDataSnippet*>  _snippetDataList;
 
+   /**
+    * _processorInfo contains the targetted hardware level for the compilation.
+    * This may be different than the real hardware the JVM is currently running
```

Comment:
Is this Java specific?  Or should "JVM" be rephrased to something runtime-neutral?